tests/manage/monitoring/prometheus/test_deployment_status.py

```diff
@@ -92,6 +95,7 @@ def test_ceph_monitor_stopped(measure_stop_ceph_mon, threading_lock):
 @pytest.mark.polarion_id("OCS-2724")
 @pytest.mark.parametrize("create_mon_quorum_loss", [True])
 @skipif_managed_service
+@skipif_ms_provider_and_consumer
 @skipif_ocs_version("<4.9")
 def test_ceph_mons_quorum_lost(measure_stop_ceph_mon, threading_lock):
```

Comment:
@fbalak Shall I skip this alerts in provider client solutions or Not. For now I added skipif marker However I am not sure on this. Could you please comment?

Reply:
I think that all those tests need to be updated but can run on provider.